frontend/src/pages/Landing/LandingPage.tsx

```diff
@@ -87,13 +89,18 @@ export const LandingPage = (): JSX.Element => {
   return (
     <>
       <AppPublicHeader />
-      <LandingSection bg="primary.100" pt={{ base: '2rem', md: 0 }}>
+      <LandingSection bg="primary.100" pt={{ base: '2rem', md: 0 }} px="0">
         <Stack
           direction={{ base: 'column', lg: 'row' }}
           align="center"
           spacing={{ base: '1.5rem', md: '3.125rem', lg: '7.5rem' }}
+          pl={{ base: '1.5rem', md: '5.5rem', lg: '9.25rem' }}
         >
-          <Flex flexDir="column" flex={1}>
+          <Flex
+            flexDir="column"
+            flex={1}
+            pr={{ base: '1.5rem', md: '5.5rem', lg: '0' }}
```

Comment:
any reason for the right padding? if not, I think we can just adjust the `Stack` parent's `spacing` prop, which adds spacing between its children.

Reply:
the right padding is used when the layout is a vertical stack (so when the layout is < `lg`). `spacing` won't work in this case since it'll be vertical spacing

didn't apply the right padding to the `Stack` parent because both its children have different right padding values - the text portion has right padding, but `LottieAnimation` is flush to the right viewport so has 0 right padding